src/components/SelectTime/SelectRegion.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const SelectRegion = () => {
+  const [checkedRegions, setCheckedRegions] = useState([]);
+
+  const handleOnClick = region => {
+    setCheckedRegions([region]);
+  };
+
+  const RegionList = ['홍대', '청담씨네시티', '목동', '피카디리1958', '신촌아트레온'];
+
+  return (
+    <>
+      <St.SelectRegion>
+        {RegionList.map(region => (
+          <St.RegionButton
+            key={region}
+            onClick={() => handleOnClick(region)}
+            isChecked={checkedRegions.includes(region)}
```

Comment:
p1) styled-components에 porps로 넘겨주려면 '$'이 키워드를 사용해야 합니다! 

(언젠가부터 바뀌었더라구요..)


```suggestion
            $isChecked={checkedRegions.includes(region)}
```

Reply:
헉 반영하겠습니다..!